src/main/java/duke/Task.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+/**
+ * Task class that creates a task inputted by user.
+ */
+public abstract class Task {
+    protected String item;
+    protected boolean done;
```

Comment:
Perhaps this should be:
```suggestion
    protected boolean isDone;
```
instead? According to [Java coding standards](https://se-education.org/guides/conventions/java/intermediate.html), boolean variables/methods should be named to sound like booleans

Reply:
Thank you for pointing it out. I've made the changes.